### ISSUE TITLE
missing dependency to urrlib3

### ISSUE THREAD

**Issue by laf0rge** [influxdata/influxdb-client-python#537]
### Specifications

* Client Version: 'pip install 'influxdb-client[ciso]' of today
* InfluxDB Version: 2.5.1
* Platform: Debian 11

### Code sample to reproduce problem

no sample code needed; this python library is missing a dependency to urllib

### Expected behavior

`pip install` should install all required upstream python modules

### Actual behavior

urllib3 is not installed automatically when installing influxdb_client:

```
  File "/usr/local/lib/python3.9/dist-packages/influxdb_client/__init__.py", line 60, in <module>
    from influxdb_client.configuration import Configuration
  File "/usr/local/lib/python3.9/dist-packages/influxdb_client/configuration.py", line 20, in <module>
    import urllib3
ModuleNotFoundError: No module named 'urllib3'
```

### Additional info

It's easy to work around and install it manually, but still normally a python module should list all of its dependencies.

**Comment by laf0rge** [comment]
The same is true for dateutil:
```
Traceback (most recent call last):
  File "/usr/local/bin/./tasmota-influxdb.py", line 10, in <module>
    from influxdb_client import InfluxDBClient
  File "/usr/local/lib/python3.9/dist-packages/influxdb_client/__init__.py", line 382, in <module>
    from influxdb_client.client.delete_api import DeleteApi
  File "/usr/local/lib/python3.9/dist-packages/influxdb_client/client/delete_api.py", line 7, in <module>
    from influxdb_client.client._base import _BaseDeleteApi
  File "/usr/local/lib/python3.9/dist-packages/influxdb_client/client/_base.py", line 17, in <module>
    from influxdb_client.client.flux_csv_parser import FluxResponseMetadataMode, FluxCsvParser, FluxSerializationMode
  File "/usr/local/lib/python3.9/dist-packages/influxdb_client/client/flux_csv_parser.py", line 12, in <module>
    from influxdb_client.client.util.date_utils import get_date_helper
  File "/usr/local/lib/python3.9/dist-packages/influxdb_client/client/util/date_utils.py", line 6, in <module>
    from dateutil import parser
ModuleNotFoundError: No module named 'dateutil'
````

**Comment by powersj** [comment]
How did you install the library? Both of these are specified in the `requires` section of the [setup.py](https://github.com/influxdata/influxdb-client-python/blob/master/setup.py#L12)?

**Comment by powersj** [comment]
```
~/test 
❯ . bin/activate
~/test via 🐍 v3.10.8 (test) 
❯ pip list
Package    Version
---------- -------
pip        22.2.2
setuptools 65.4.0
wheel      0.37.1

[notice] A new release of pip available: 22.2.2 -> 22.3.1
[notice] To update, run: pip install --upgrade pip
 pip install influxdb-client[ciso]
Collecting influxdb-client[ciso]
  Downloading influxdb_client-1.35.0-py3-none-any.whl (719 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 720.0/720.0 kB 2.9 MB/s eta 0:00:00
Collecting certifi>=14.05.14
  Using cached certifi-2022.9.24-py3-none-any.whl (161 kB)
Requirement already satisfied: setuptools>=21.0.0 in ./lib/python3.10/site-packages (from influxdb-client[ciso]) (65.4.0)
Collecting python-dateutil>=2.5.3
  Using cached python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
Collecting reactivex>=4.0.4
  Using cached reactivex-4.0.4-py3-none-any.whl (217 kB)
Collecting urllib3>=1.26.0
  Downloading urllib3-1.26.13-py2.py3-none-any.whl (140 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 140.6/140.6 kB 22.7 MB/s eta 0:00:00
Collecting ciso8601>=2.1.1
  Using cached ciso8601-2.2.0-cp310-cp310-linux_x86_64.whl
Collecting six>=1.5
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting typing-extensions<5.0.0,>=4.1.1
  Downloading typing_extensions-4.4.0-py3-none-any.whl (26 kB)
Installing collected packages: ciso8601, urllib3, typing-extensions, six, certifi, reactivex, python-dateutil, influxdb-client
Successfully installed certifi-2022.9.24 ciso8601-2.2.0 influxdb-client-1.35.0 python-dateutil-2.8.2 reactivex-4.0.4 six-1.16.0 typing-extensions-4.4.0 urllib3-1.26.13

[notice] A new release of pip available: 22.2.2 -> 22.3.1
[notice] To update, run: pip install --upgrade pip
~/test via 🐍 v3.10.8 (test) took 2s 
❯ pip list
Package           Version
----------------- ---------
certifi           2022.9.24
ciso8601          2.2.0
influxdb-client   1.35.0
pip               22.2.2
python-dateutil   2.8.2
reactivex         4.0.4
setuptools        65.4.0
six               1.16.0
typing_extensions 4.4.0
urllib3           1.26.13
wheel             0.37.1

[notice] A new release of pip available: 22.2.2 -> 22.3.1
[notice] To update, run: pip install --upgrade pip

```

**Comment by laf0rge** [comment]
I definitely did install it with "pip install 'influxdb-client[ciso]". But I did install/remove some other packages in between the "pip install" of influxdb_client.  So what *might* have happened is the following:
* installing some debian package with dependency to (debian packaged) urllib3
* doing the "pip install influxdb-client[ciso]"
* removing some debian package which had the depdency to (debian packaged) urllib3, plus "apt autoremove"
* now the python depenency is of course no longer fulfilled

Sorry for what was probably a false alarm.
